app/api/db/models/factories.py

```diff
@@ -57,19 +58,25 @@ class Meta:
         sqlalchemy_session_persistence = "commit"
```

Comment:
i googled `sqlalchemymodelfactory` but can't find anything on that. is that part of sqlalchemy, a library, or something we made?

Reply:
A part of factoryboy
https://factoryboy.readthedocs.io/en/stable/orms.html#sqlalchemy